bson/bsoncodec/bsoncodec.go

```diff
@@ -119,38 +119,159 @@ func (vde ValueDecoderError) Error() string {
 // value.
 type EncodeContext struct {
 	*Registry
+
+	// MinSize causes the Encoder to marshal Go integer values (int, int8, int16, int32, int64,
+	// uint, uint8, uint16, uint32, or uint64) as the minimum BSON int size (either 32 or 64 bits)
+	// that can represent the integer value.
+	//
+	// Deprecated: Use bson.Encoder.IntMinSize instead.
```

Comment:
Good question. There historically hasn't been a standardize doc link format, so we've bounced between formats. However, I did some digging and it turns out Go comments now support [doc links](https://tip.golang.org/doc/comment#doclinks)!

Using those doc links, I believe the format would be
```
[bson.Encoder.IntMinSize]
```
However, since `bsoncodec` doesn't import `bson`, I'm not sure if the docs generator would be able to understand that link. I'll see if I can get doc links working. If so, I'll update all of the deprecation messages with a doc link.

Reply:
Updated the PR with doc links.